src/hooks/Circles/usePrivatePosts.ts

```diff
@@ -156,84 +148,6 @@ export function usePollPageInfoForUsers(
   });
 }
 
-export const useLookupUsers = (
```

Comment:
These are hooks cobbled together to make up for the fact we didn't have backend support for "Conversations" and are no longer needed.